lib/spack/docs/getting_started.rst

```diff
@@ -600,6 +600,61 @@ flags to the ``icc`` command:
+They are usually distributed with `oneAPI <https://software.intel.com/content/www/us/en/develop/articles/oneapi-repo-instructions.html>`_ and compilers are activated with ``source /opt/intel/inteloneapi/setvars.sh`` on Linux.
+
+One can record these activation steps by comparing set environment variables:
+
+.. code-block:: console
+
+   env > env.txt
+   set +e
+   source /opt/intel/inteloneapi/setvars.sh
+   env > env_extras.txt
+   diff env.txt env_extras.txt
+
+All newly set environment hints will then be shown in this diff.
+A typical entry in ``compilers.yaml`` looks like this:
+
+.. code-block:: yaml
+
+   # DPC++ 2021.1-beta07 (2020.5.0.0604)
+   - compiler:
+       spec: clang@11.0.0
+       paths:
+         cc: /opt/intel/inteloneapi/compiler/latest/linux/bin/clang
+         cxx: /opt/intel/inteloneapi/compiler/latest/linux/bin/dpcpp
+         f77:
+         fc:
+       flags: {}
+       operating_system: ubuntu18.04
+       target: x86_64
+       modules: []
+       environment:
+         set:
+           OCL_ICD_FILENAMES: libintelocl_emu.so:libalteracl.so:/opt/intel/inteloneapi/compiler/latest/linux/lib/x64/libintelocl.so
+           ACL_BOARD_VENDOR_PATH: /opt/Intel/OpenCLFPGA/oneAPI/Boards
+           FI_PROVIDER_PATH: /opt/intel/inteloneapi/mpi/2021.1-beta07//libfabric/lib/prov
+           CLASSPATH: /opt/intel/inteloneapi/mpi/2021.1-beta07//lib/mpi.jar
+           INTELFPGAOCLSDKROOT: /opt/intel/inteloneapi/compiler/2021.1-beta07/linux/lib/oclfpga
+           NLSPATH: /opt/intel/inteloneapi/mkl/2021.1-beta07/lib/intel64/locale/%l_%t/%N
+           DPCPP_ROOT: /opt/intel/inteloneapi
+           ONEAPI_ROOT: /opt/intel/inteloneapi
+           MKLROOT: /opt/intel/inteloneapi/mkl/2021.1-beta07
+           TBBROOT: /opt/intel/inteloneapi/tbb/2021.1-beta07/env/..
+           I_MPI_ROOT: /opt/intel/inteloneapi/mpi/2021.1-beta07
+         prepend_path:
+           LD_LIBRARY_PATH: /opt/intel/inteloneapi/tbb/2021.1-beta07/env/../lib/intel64/gcc4.8:/opt/intel/inteloneapi/mpi/2021.1-beta07//libfabric/lib:/opt/intel/inteloneapi/mpi/2021.1-beta07//lib/release:/opt/intel/inteloneapi/mpi/2021.1-beta07//lib:/opt/intel/inteloneapi/mkl/2021.1-beta07/lib/intel64:/opt/intel/inteloneapi/compiler/latest/linux/lib:/opt/intel/inteloneapi/compiler/latest/linux/lib/x64:/opt/intel/inteloneapi/compiler/2021.1-beta07/linux/lib/oclfpga/host/linux64/lib:/opt/intel/inteloneapi/compiler/2021.1-beta07/linux/lib/oclfpga/linux64/lib:/opt/intel/inteloneapi/compiler/latest/linux/compiler/lib/intel64_lin:/opt/intel/inteloneapi/compiler/latest/linux/compiler/lib
+           CPATH: /opt/intel/inteloneapi/tbb/2021.1-beta07/env/../include:/opt/intel/inteloneapi/mpi/2021.1-beta07//include:/opt/intel/inteloneapi/mkl/2021.1-beta07/include:/opt/intel/inteloneapi/dev-utilities/2021.1-beta07/include:/opt/intel/inteloneapi/compiler/latest/linux/include
```

Comment:
Again, this is including MPI information that definitely should not be part of the compiler.